src/Core/Utilities/MultipartRequestHelper.cs

```diff
@@ -0,0 +1,29 @@
+using System;
+using System.IO;
+using System.Linq;
+
+namespace HotChocolate.Utilities
+{
+
+    public static class MultipartRequestHelper
+    {
+        // Content-Type: multipart/form-data; boundary="----WebKitFormBoundarymx2fSWqWSd0OxQqq"
+        // The spec says 70 characters is a reasonable limit.
+        public static string GetBoundary(string contentType)
+        {
+            var boundary = contentType
```

Comment:
We cannot introduce new string splitting in the request. We need a real parser for the request, I will look at this since I am working on the new JSON request parser.